### ISSUE TITLE
example html report issues

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion#1326]
Hi,
couple of weird things and my thoughts on [html report](http://scipion.cnb.csic.es/scipionbox/lastHTMLReport/). As I understand this is still work in progress. So let's progress it :)
1) ctf is determined wrong in almost all cases - I think a wrong range of defocuses is used
2) on pages 7,8,9 all thumbnails are missing:
![screenshot_20171111_170555](https://user-images.githubusercontent.com/6952870/32691693-7c698478-c703-11e7-9d00-b091f8577e50.png)
3) https://github.com/I2PC/scipion/wiki/Streaming-Processing here in the screenshot of the demo spherical aberration is 7.08 so as pixel size. Should be 2.07 I guess..
4) is it just my Chrome browser or the whole table is "shifted":
![screenshot_20171111_171107](https://user-images.githubusercontent.com/6952870/32691687-5eeb8dce-c703-11e7-9f85-fb9b2d8a523c.png)
5) just my personal opinion here: resolution and defocus coverage plots would look more natural if they were vertical?
6) I would advocate for having ALL values 0.2f format (resolution 6.02A, defocus 1.52um (please in microns, not Angstroms!!), ratio 1.25, astigmatism ..) for all tables/plots so it would be easy to comprehend
7) movie gain monitor doesn't show any plots - I guess it's not running?
8) upon displaying the thumbnails, there is a very long filename in the header, like `imgMicThumbs/FoilHole_20524073_Data_20525427_20525428_20171021_0118_Fractions_aligned_mic.png`, which doesn't even fit into thumbnail ;) Could this be shortened at least to `FoilHole_20524073_Data_20525427_20525428_20171021_0118` ?
![screenshot_20171111_172305](https://user-images.githubusercontent.com/6952870/32691797-139917a4-c705-11e7-8147-5b008df9b39e.png)
9) in global shifts plot frame numbers are often overlaid on top of each other, I guess we could put a label only for the first and the last frames?
![screenshot_20171111_172713](https://user-images.githubusercontent.com/6952870/32691825-92f464cc-c705-11e7-9a80-586b07bf6c4a.png)

**Comment by pconesa** [comment]
@azazellochg thanks for the feedback...let me clarify some points.

1.- This is mainly my bad for choosing the wrong data set (jmbFalconMovies) that renders that result.
2.- Yep, that is right cause I have troubles when running the protocols, but the HTML report should be fine, only that run failed cause I did not have eman installed (which is used for the thumbnails). 
3.- My bad, I think I enter the wrong value.
4.- That is a tricky one, especially if so resize the browser, that JS component does not behave always right. We are assuming that "feature".
5.- Not the first one to suggest that, thanks.
6.- Make sense, thanks
7.- Gain....I'll have a look it should show 1 every 500 movies, which for the dataset, means one. Useless as a demo.
8.- Certainly look awful, we can try even something shorter, not sure what.
9.- Here we are using what the protocol has generated, so it' more a protocol issue, but make also sense.

In summary....1,2,3,7 are more a "demo" issue.

4, 5, 6, 8 are HTML ones

9 is more on the protocol side but would be nice to solve it anyway.

This has been split in #1332, #1333, #1334
